src/Symfony/Component/Validator/Resources/translations/validators.hr.xlf

```diff
@@ -220,7 +220,7 @@
             </trans-unit>
             <trans-unit id="58">
                 <source>Unsupported card type or invalid card number.</source>
-                <target>Neispravan broj kartice ili tip kartice nije podržan.</target>
+                <target>Tip kartice nije podržan ili neispravan broj kartice.</target>
```

Comment:
@toniperic "Tip kartice nije podržan ili **je** neispravan broj kartice." Even better: "Tip kartice nije podržan ili je broj kartice neispravan."

Reply:
@HypeMC agree. The latter seems perfect.